app.rb

```diff
@@ -1,19 +1,12 @@
-require './person'
-require './book'
-require './student'
-require './classroom'
-require './rental'
-require './teacher'
-class App
-  def initialize
-    @books = []
-    @students = []
-    @teachers = []
-    @people = []
-    @rentals = []
-  end
+require_relative 'person'
+require_relative 'book'
+require_relative 'student'
+require_relative 'teacher'
+require_relative 'rental'
+require 'json'
 
-  # list of books
+# Module for managing books
+module BookManagement
   def list_books
     if @books.empty?
       puts 'No books found.'
```

Comment:
- [ ] When I create a book as seen above and I exit the app, when I try to list the books in the json file, the app crashes. Kindly fix this.

  ![image](https://github.com/AlyAtalla/oop-library/assets/81468675/4882b798-59ac-4617-a668-8590b33bff33)

- [ ] Kindly ensure your app works fine before re-submitting.